components/channel_header_dropdown/menu_items/toggle_mute_channel/toggle_mute_channel.js

```diff
@@ -73,7 +72,7 @@ export default class MenuItemToggleMuteChannel extends React.PureComponent {
         return (
             <Menu.ItemAction
                 id={id}
-                show={channel.type !== Constants.DM_CHANNEL}
+                show={true}
```

Comment:
You can actually just remove the show prop completely here. You don't need to explicitly set it to true since the `defaultProps` of `ItemAction` has show already set to true.
```suggestion
```

Reply:
There will be a linter error. `error  Value must be set for boolean attributes  react/jsx-boolean-value` in whole project boolean value is not omitted

Reply:
@nadalfederer 
You would get this error if you have the following  
```
<Menu.ItemAction 
     id={id} 
     show 
     onClick={this.handleClick} 
     text={text} 
/>
```
However, what I mean is to completely remove it as a prop since it already has a defaultProp of true. So you can do the following to get rid of the error you mentioned
```
<Menu.ItemAction 
     id={id} 
     onClick={this.handleClick} 
     text={text} 
/>
```
Notice how I am not passing in the show prop anymore. Hopefully this helps! Thanks 😄